### DISCUSSION TITLE
Cross Probing iModel problem

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#4518] [General] [answered]
I am implementing cross probing in our application.  This is exactly the same capability as in the Cross-Probing example in the sample showcase.  

For this I have two dgns files, one for the 3D model of the plant and second for the 2D PFD.  Both models were imported in a dgn which is then uploaded in iModel hub using iTwinSynchronizer.  When I open the iModel in Design Review, it seems the 2D PFD model is not uploaded.

I found another (unrelated) 2D dgn and for testing added this model to my dgn.  When I upload this dgn, I can see the second 2D model (called Civil) in Design Review but the PFD model is still not uploaded.  No errors were reported by iTwinSynchronizer.

The dgn file is attached.  It has 3 models

$Classifier -> 3D model that we use as classifier
PDF -> 2D PFD we want to use for cross probing -> this 2D model is not uploaded
Civil -> a 2D drawing added in the model for test purposes -> this 2D model is uploaded and can be viewed in Design Review

![image](https://user-images.githubusercontent.com/83007711/196961982-1abdb6e1-23ff-426e-a4d2-a2a513902cb3.png)

I looked at the Properties of the two 2D models in MicroStation but I didn't see any difference in their parameter - as far as I can tell, they are setup/configured the same in the way (I am not MS expert, so may be missing something).  

Can you please guide what might be the problem with the PFD model and why it is not being uploaded to iModelHub.

[Beach-Master-2.zip](https://github.com/iTwin/itwinjs-core/files/9830646/Beach-Master-2.zip)
.
If you wish to look at the uploaded iModel in iModelHub, here are the details

Project: d20678d8-ad8c-4732-a32f-5ec62e2befc0 (Co Survey Digital Twin)
iModel: f52414b7-4704-44f0-a3fe-38a293942c24  (Beach_2)
ChangeSet: eeb50c4f6e6f482ab33ca8b48484465f0575673a (New PFD added - Data removed)

Thank you,

Ovais

**Comment by aruniverse** [comment, marked as answer]
@jffmarker

**Reply by jffmarker** [reply]
I've asked someone on the MicroStation Connector team to take a look; will let you know what they find.

**Reply by cammac1803** [reply]
The PFD model has a GCS attached to it. As such, the connector treats it as a design model (that needs to be referenced in to appear) and not a drawing model. If you remove the GCS from the model and synchronize again, it should work as you expect.

**Reply by arnab-git-hub** [reply]
@ovaisa Can you please let us know if the suggested workflow worked for you? If yes, can you please mark it as answered.
